lib/extract/Makefile

```diff
@@ -52,16 +52,29 @@ PSYDATA_LIB_DIR ?= ./..
 # the listed data types and scalar and 2-dimensional arrays. Note that
 # domain-specific implementation will create their own version with different
 # number of dimensions if required.
-PROCESS_ARGS = -types=char,int,long,logical,real,double -dims=2
+PROCESS_ARGS = -types=char,int,long,logical,real,double -dims=1,2,3,4
 PROCESS = $$($(PSYDATA_LIB_DIR)/get_python.sh) $(PSYDATA_LIB_DIR)/process.py
 
-.PHONY: all compare
+.PHONY: all compare test
 	
 compare: compare_variables_mod.o
 
+test_compare: compare_variables_mod.o test_compare.o
```

Comment:
Oh and gitignore the generated test_compare binary

Reply:
Done.